test/posts.test.js

```
@@ -24,8 +25,12 @@ describe('test /posts endpoint', () => {
     };
   });
 
-  beforeAll(() => {
-    Promise.all(posts.map(post => addPost(post)));
+  beforeAll(async () => {
+    try {
```

Don't bother with try/catch here.  Let it fail loudly in tests.